docs/commands.rst

```diff
@@ -269,6 +269,12 @@ Examples of patterns are available in ``tests/OK/patterns.lp``.
 
 The ``notation`` command is used to specify a notation for a symbol.
 
+When declared as notations, identifiers then must be used at correct places
+and as such cannot make valid terms on their own anymore.
+To reaccess the value of the identifier without the notation properties,
+wrap it in parentheses.
+
```

Comment:
@yannl35133 Would you mind to add an example on line 286?
```
// + is not a valid term since, as it is declared as infix,
// the system expects + to only appear in expressions of the form x+y
// you can get around this by using (+) instead
```